editoast/core_client/src/lib.rs

```diff
@@ -23,6 +23,30 @@ use tracing::trace;
 
 pub use mq_client::RabbitMQClient;
 
+/// An enum to wrap RabbitMQ's routing keys
+///
+/// - [WorkerKey::Composition]: worker key based on two ids, most likely an infra id and a timetable id
+/// - [WorkerKey::Infra]: worker key based on an infra id
+/// - [WorkerKey::Empty]: empty worker key
+#[derive(Debug)]
+pub enum WorkerKey {
+    Composition(i64, i64),
```

Comment:
The worker key is actually interpreted by Core on spawn to extract the IDs. It's not just an abstract hash. It could be better to directly link the variant to its meaning.


```suggestion
    Timetable { timetable_id: i64, infra_id: i64 },
```

Reply:
I was not sure if the ids were always the infra and timetable ids but since it is the case, I strongly agree

Reply:
Done